docs/development_suite/api-console/api-design/microservice-containers.md

```diff
@@ -0,0 +1,88 @@
+
+## Sidecar Containers
+
+For several reasons although, you may want to extend the main container functionalities **without modifying** the business logic of your application.
+
+To better understand this need, let's imagine this common scenario.  
+You have a core microservice that processes user orders. This microservice is responsible for validating the order, calculating the total amount, and then sending it to the fulfillment center.
+
+However, you now have a new requirement: adding a logging functionality to your microservice to monitor every order that is being processed.  
+You could simply add the logging code to the core service source code, but sometimes this option is not viable, to name a few reasons:
+- the service may be maintained by another team
+- modifying the legacy core service may introduce errors
+
+For these reasons the sidecar-pattern comes very handy. You can develop a lightweight service that specializes in logging requests to the main application and store these logs in a centralized logging system.  
+You can then deploy this logging service along with the main application on the same host. This ensures that the sidecar and main service containers are always running on the same machine.
+
+In this section we will go deeper in how you can manage this kind of containers for your microservices.
+
+
+## Microservice detail page
+
+The containers table in the microservice detail page gives an immediate overview of all the containers that your microservice is made of, indicating which is the **main container** and which is the **sidecar** one.
+
+The table shows a summary of the containers resource request and limits for CPU and memory as well as which configmaps and secrets are used by which container. From here is also possible to add and delete sidecars to the current microservice.
+
+![Microservice Containers](img/microservice-containers/containers-table.png)
+
+### Exposing a specific container
+
+:::info
+Exposing a container means making it reachable from other workloads inside the same Project environment.
+
+By default all the containers that specify a container port are exposed by the Console if no other changes are made.
+:::
+
+From the containers table you can decide whether to expose all your service containers or toggle only the ones you want to expose. In the example below, only the **main container** is configured to be exposed.
+
+![Exposing containers](img/microservice-containers/exposing-containers.png)
+
+### Sidecar detail
+
+Clicking on the sidecar name in the table, you enter the sidecar configuration page.  From this page you can inspect and update the configuration as well as adding configMaps, secrets, update the docker image and all the other settings, the same way as you can edit a microservice configuration.
```

Comment:
here we could cross link to the Sidecar detail doc page

Reply:
you mean this very same page?

Reply:
yes like anchor to the specific section

Reply:
already present too, there's the link to each section of the microservice detail